Prevent the use of the hidden attribute for symbols. In particular,
the attribute must not be applied to '__libc'-prefixed symbols to allow
overwriting the dummies defined in dummies.cc by other non-weak
implementations such as those in 'libc_noux.lib.so'.

This attribute is applied inconsistently in
libc/lib/libc/include/libc_private.h anyway.

--- src/lib/libc/sys/sys/cdefs.h
+++ src/lib/libc/sys/sys/cdefs.h
@@ -447,7 +447,7 @@
 #if __GNUC_PREREQ__(4, 0)
 #define	__null_sentinel	__attribute__((__sentinel__))
 #define	__exported	__attribute__((__visibility__("default")))
-#define	__hidden	__attribute__((__visibility__("hidden")))
+#define	__hidden
 #else
 #define	__null_sentinel
 #define	__exported
